<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="productView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="editSpinBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>172</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addProductButton">
       <property name="text">
        <string>Добавить товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editProductButton">
       <property name="text">
        <string>Редактировать товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteProductButton">
       <property name="text">
        <string>Удалить товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="text">
        <string>Отсортировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expectedProductButton">
       <property name="text">
        <string>Отсутствующие товары</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="deleteSpinBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsProductButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>252</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Товары с параметрами</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="settings">
     <item>
      <layout class="QVBoxLayout" name="settingsText1">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Кол-во от</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Цена от </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Хранятся от</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settingsSpb1">
       <item>
        <widget class="QSpinBox" name="amountFromSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="priceFromSpinBox">
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="storedFromSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settingsText2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>до</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>до</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>до</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settingsSpb2">
       <item>
        <widget class="QSpinBox" name="amountToSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="priceToSpinBox">
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="storedToSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="checkBoxLayout">
       <item>
        <widget class="QCheckBox" name="amountCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="priceCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="storedCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>285</y>
      <width>252</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="aboutProductButton">
       <property name="text">
        <string>Информация о товаре: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="aboutProductLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
